cvx/linalg/random.py

```diff
@@ -1,7 +1,9 @@
 # -*- coding: utf-8 -*-
 import numpy as np
 
+from cvx.linalg.types import Matrix
 
-def rand_cov(n):
+
+def rand_cov(n: int) -> Matrix:
     a = np.random.randn(n, n)
-    return np.transpose(a) @ a
+    return np.array(np.transpose(a) @ (a))
```

Comment:
That was a matrix before, no?

Reply:
I am not sure how to explicitly enforce it's that type. np.transpose(a) @ a gives some sort of Any

Reply:
I see. We can also override mypy by ignoring individual lines if we know they are doing the right thing by adding `# type:ignore` as a comment.